src/wowman/core.clj

```diff
@@ -41,9 +41,11 @@
 
 (def colours (utils/nav-map-fn -colour-map))
 
-(defn paths
-  "returns a map of paths whose location may vary depending on the location of the current working directory"
-  [& path]
+(defn generate-path-map
+  "generates a map of filesystem paths whose location may vary according to the current working directory and environment variables.
+  this map of paths is generated during init and is then fixed in application state.
+  ensure the correct environment variables and cwd are set prior to init to ensure isolation during tests"
```

Comment:
"ensure the correct environment variables and cwd are set prior to init for proper isolation during tests"